Lib/selectors.py

```diff
@@ -348,18 +348,15 @@ def __init__(self):
         super().__init__()
         self._selector = self._selector_cls()
 
-    def register(self, fileobj, events, data=None, *, exclusive=False):
+    def register(self, fileobj, events, data=None, *, extra_events=False):
```

Comment:
`extra_events=0` maybe?

Reply:
I prefer `None`, but no `False` plz.

Reply:
`None`?

Reply:
`extra_events` is a bitmask, why `None`?

Reply:
> `extra_events` is a bitmask, why `None`?

In the most circumstance, people do not need the extra_events , so I give it a default value

Reply:
With `0` default value people still don't need to pass `extra_events`.
But we don't need to extra check the value for `None`, `0` should work fine as *no flags* for all possible selectors: `poll`, `epoll`, `kqueue` etc

Reply:
that sounds a good idea. I will fix it! Thx!